mkt/lookup/helpers.py

```diff
@@ -41,6 +41,20 @@ def app_header(context, app, page_type=''):
                                    'is_operator': is_operator}))
 
 
+# page_type is used for setting the link 'sel' class
+@register.function
+@jinja2.contextfunction
+def website_header(context, website, page_type=''):
+    t = env.get_template('lookup/helpers/website_header.html')
+
+    is_admin = acl.action_allowed(context['request'], 'Users', 'Edit')
+    is_staff = acl.action_allowed(context['request'], 'Apps', 'Configure')
```

Comment:
depends what is_staff will be used for, but testing for 'Apps:Configure' seems wrong for websites.